apps/src/lib/util/audioApiDropletConfig.js

```diff
@@ -32,6 +33,19 @@ const dropletConfig = {
       0: () => getAssetDropdown('audio')
     },
     assetTooltip: {0: chooseAsset.bind(null, 'audio')}
+  },
+  playSpeech: {
+    func: 'playSpeech',
+    parent: executors,
+    paramButtons: {minArgs: 2, maxArgs: 3},
+    paletteParams: ['text', 'gender', 'language'],
+    params: ['"Hello World!"', '"female"', '"en-US"'],
+    dropdown: {
+      1: ['"female"', '"male"'],
+      2: Object.keys(appOptions.azureSpeechServiceLanguages).map(x => `"${x}"`)
```

Comment:
What is `x` in this case? Give it a more descriptive name like `language` or `key` or `locale`